src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <main class="calendar calendar--start-day--sun calendar--month-length--29">
+      <div class="calendar__cell calendar__cell--1"></div>
```

Comment:
it does not need to be written in the class
calendar__cell--1
it needs to be created in a loop